WalletWasabi.Gui/Config.cs

```diff
@@ -67,6 +67,15 @@ public class Config : IConfig
 		[JsonProperty(PropertyName = "RegTestBitcoinCorePort")]
 		public int? RegTestBitcoinCorePort { get; internal set; }
 
+		[JsonProperty(PropertyName = "MainNetFetchFromLocalOnly")]
+		public bool MainNetFetchFromLocalOnly { get; internal set; }
+
+		[JsonProperty(PropertyName = "TestNetFetchFromLocalOnly")]
+		public bool TestNetFetchFromLocalOnly { get; internal set; }
+
+		[JsonProperty(PropertyName = "RegTestFetchFromLocalOnly")]
+		public bool RegTestFetchFromLocalOnly { get; internal set; }
```

Comment:
frankly, I think a separate type `NetworkSettings` should encapsulate it and having config like:

```csharp
{
 "mainnet" : { "host":"1.2.3.4", "port":2939, "fetchLocally" : true },
 "testnet" : { "host":"1.2.3.4", "port":2939, "fetchLocally" : true },...
}
```

Maybe can refactor RegTestBitcoinCorePort/Endpoint later on to fix this, but I think it would be great to start with fetchLocally.